cmd/veneur/main.go

```diff
@@ -120,9 +117,7 @@ func main() {
 			}).Error("Error reading from UDP")
 			continue
 		}
-		select {
-		case parserChan <- buf[:n]:
-		}
+		parserChan <- buf[:n] // TODO: termination condition for this channel?
```

Comment:
Eager for some feedback on this. I wrote it this way because [the docs](https://gobyexample.com/non-blocking-channel-operations) said this was how you do a non-blocking write. Is that not a good pattern / does it not make any difference? I thought it might eek out a bit more. ;)